draft-ietf-quic-multipath.md

```diff
@@ -578,39 +550,39 @@ Path Identifiers.
 ### Effect of MP_RETIRE_CONNECTION_ID Frame {#retire-cid-close}
 
 Receiving a MP_RETIRE_CONNECTION_ID frame causes an endpoint to discard
-the resources associated with that Connection ID. Note that retirement of
-Connection IDs will not effect the use of Path Identifier for the specific path.
-The list of received packets used to send acknowledgements is also remain
+the resources associated with that connection ID. Note that retirement of
+connection IDs will not effect the use of Path Identifier for the specific path.
+The list of received packets used to send acknowledgements also remains
 uneffected as the packet number space is associated with a path.
```

Comment:
I find this confusing, and I'm surprised one would even consider this. This is just plain RFC 9000 behavior.

Reply:
in any case, "uneffected" --> "unaffected".